.github/workflows/cd_ios.yml

```diff
@@ -43,6 +43,48 @@ jobs:
           cd examples/mobile/ios && bundle install
           bundle exec fastlane build_and_deploy_to_firebase --env enterprise build_number:$GITHUB_RUN_NUMBER
 
+  Build-And-Deploy-To-TestFlight-InternalTest:
```

Comment:
Shouldn't we only to trigger internal release on tags?

Reply:
Do we need one as there is already a build existing for the same commit on `main` ?

Reply:
Actually I just rechecked the condition:

Build triggers are like below:

Every commits on `main` - `Firebase build, Internal TestFlight build`
Every `tag` pushed - `Firebase build, Internal TestFlight build & AppStore Release build.`

Reply:
please correct me if I'm wrong, my understanding is that we only need the internal builds for stable versions, and public builds for each PR merged.

Reply:
Every commits on main - Firebase build
Every tag pushed - Internal (TestFlight & Google Play) build & Public (TestFlight & Google Play) Release build.

Reply:
Cool. Thanks for correcting guys. Fixed the triggers.